culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+This post proposes a system of **trust levels** for SourceCred the
+system, which describe how we intend to handle the very real questions
+of adversarial actors and gaming resistance over different time scales.
+
+The ideas proposed herein have been discussed informally within our
+community, but with somewhat confusingly varying semantics. It may help
+to have this shared reference to point to.
+
+**Note:** The term *trust levels* as used in this post is largely
+separate from [Discourse trust levels][discourse-trust], which are a
+construct of our forum software and not related to the SourceCred
+protocol in any way.
+
+[discourse-trust]: https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/
+
+# Motivation
+
+To achieve its goals when applied to real-world projects, SourceCred
+will need to be resistant to gaming by people who explicitly (but
+perhaps covertly) attempt to maximize their short-term or long-term
+profits at all other costs. This is because SourceCred is intended to be
+used to allocate scarce resources, and so actors will have incentives to
+acquire those resources and may have no natural incentive to contribute
+back to the community or behave in accordance with our preferred norms.
+
+However, full resistance to adversarial behavior will require
+substantial research, design, and implementation effort. Some of this
+effort cannot even begin until we have a fairly stable understanding of
+other parts of the pipeline. It doesn’t make sense to discuss defenses
+until we’ve defined the protocol in which everyone is to operate.
+
+Furthermore, while the effort required to improve gaming resistance will
+surely bring us benefits in the long term, those benefits only
+materialize when they actually prevent gaming, which can in turn only
+occur if people are trying to game the system in ways that our existing
+systems cannot sufficiently mitigate. Thus, as long as our community
+remains small enough that manual chastising and blacklisting can rein in
+the renegades, we’ll receive no short-term reward for any effort spent
+on this. We have *plenty* of other initiatives into which we’d prefer to
+invest our energy.
```

Comment:
There should probably be some policy to be able to retroactively go back and "undo" some gaming that was discovered after the fact. I don't know how that would work, but it seems like a card that should be pullable.

Reply:
One can always go back and change the scores (e.g. by deleting content), assuming the community is not fully decentralized, however Grain already distributed (e.g. as an ERC-20 token) cannot (and shouldn't be) be clawed back.